.gitconfig.khan

```diff
@@ -27,6 +27,7 @@
   rb = !git review-branch
   pb = !git review-branch       ; short for 'phabricator branch'
   pr = !git pull-request
+  prs = !git pull-requests
```

Comment:
TIL about the `!` prefix for git aliases. https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases